### ISSUE TITLE
Dividend - ex date, payment day, amount per share

### ISSUE THREAD

**Issue by danielldomanski** [iprak/yahoofinance#64]
Is it possible to have data for dividends? payment day at least?

it would be nice to hear on my daily welcome routine information that some company is paying me today:) Avanza has those data, but there is only a limited stock market (the US and only some European countries ) - Yahoo has more stocks

**Comment by iprak** [comment]
I do not see any such data being returned by yahoo endpoint. I checked a few symbols and while some symbols displayed `Last Dividend` on Yahoo Finance page, the API returned no such data. I did not note payment date on Yahoo Finance site.

**Comment by danielldomanski** [comment]
Pity - on main page of stock - there are earnings date and forward dividend - I thought u can use it
![image](https://user-images.githubusercontent.com/73316336/146769644-d698c3aa-b45c-4b7c-af26-c68f5da55ae6.png)

**Comment by danielldomanski** [comment]
anyway - u did greate job:)

**Comment by iprak** [comment]
I looked at the received data again and the data element to dividend date was `"dividendDate":1636588800` which translated to 11/10/2021, 6:00:00 PM. Repeating the same exercise for MSFT gave dividendDate of 1646870400 => 3/9/2022, 6:00:00 PM but Ex-Dividend Date is Feb 16, 2022. Definitely the Ex-DividendDate is comiing from somewhere else.

By comparing against NASDAQ data, it seems that dividendDate is `Dividend Pay Date` which was 11/11/21 for AAPL (probably the next day). For MSFT, it was 3/10/2022 (again the day after 1646870400). I verified the same for a couple other symbols. So at the least I can add that.
